src/FixerDefinition/FixerDefinitionInterface.php

```diff
@@ -35,7 +35,7 @@ public function getDescription();
     public function getConfigurationDescription();
 
     /**
-     * @return null|array $defaultConfiguration     null for non-configurable fixer
```

Comment:
Depends. PHP allows methods to not actually declare their parameters. It is possible that someone might write doc like this, and then access the bound parameters using func_get_args.